pages/linux/bmon.md

```diff
@@ -0,0 +1,19 @@
+# bmon
+
+> Monitor bandwidth and capture network related statistics.
+
+- Display all interfaces:
+
+`bmon -a`
+
+- Display rates in bits per second:
+
+`bmon -b`
+
+- Set policy to define which network interfaces are displayed:
+
+`bmon -p {{interfaces}}`
+
+- Set interval in which rate per counter is calculated:
+
+`bmon -R {{rate-interval}}`
```

Comment:
Would this be in seconds (e.g. `5`), or does the interval have to be specified (e.g. `5s`)? Perhaps this could be clarified.

Reply:
Updated for clarity